src/core/server/status/plugins_status.test.ts

```diff
@@ -161,13 +161,13 @@ describe('PluginStatusService', () => {
         },
         b: {
           level: ServiceStatusLevels.degraded,
-          summary: '[2] services are degraded',
+          summary: '[savedObjects]: savedObjects degraded',
```

Comment:
This is the very slight behavior change mentioned above. If a plugin's dependencies have no custom statuses, then we use the default Core status, losing the count of dependencies that are having issues. This information didn't seem very useful in the default case anyways, so I think this is acceptable.